pkg/mons/restore_quorum.go

```diff
@@ -279,13 +279,13 @@ func getMonDetails(goodMon string, monEndpoints []string) ([]string, string, str
 func promptToContinueOrCancel(answer string) (string, error) {
 	var ROOK_PLUGIN_SKIP_PROMPTS string
 	_, ok := os.LookupEnv(ROOK_PLUGIN_SKIP_PROMPTS)
-	if ok {
+	if !ok {
 		if answer == "yes-really-restore" {
 			return "proceeding", nil
 		} else if answer == "" {
 			return "continuing", nil
```

Comment:
If the answer is blank we don't want to continue. They have to type the string to continue.

Reply:
done